tests/test_framework_model_sync.py

```
@@ -0,0 +1,328 @@
+    result = patch_hermes_default(HERMES_YAML, "new-model")
+    assert "base_url: http://127.0.0.1:4000/v1" in result
+    assert "provider: custom" in result
+    assert "memory_enabled: true" in result
+    assert "- hermes-cli" in result
+    assert "api_key: sk-x" in result
+
+
+def test_patch_hermes_default_does_not_duplicate_lines():
+    result = patch_hermes_default(HERMES_YAML, "new-model")
+    # Only one default: line should exist
+    default_lines = [l for l in result.splitlines() if "default:" in l]
```

Ruff (0.15.15)

[error] Ambiguous variable name: `l`

(E741)